core/XH.js

```diff
@@ -329,6 +329,17 @@ class XHClass {
         return this.acm.messageSourceModel.confirm(config);
     }
 
+    /**
+     * Show a modal 'prompt' dialog with a default TextInput, message and default 'OK'/'Cancel' buttons.
+     * Can provide a custom input control with the input prop (true to use default TextInput).
+     *
+     * @param {Object} config - see XH.message() for available options.
```

Comment:
Need to actually enhance those docs for XH.message.  That's the main entry point for users/doc readers....